charts/ditto/values.yaml

```diff
@@ -85,6 +86,12 @@ openshift:
 ## ----------------------------------------------------------------------------
 ## global configuration shared by all components
 global:
+  # default basic auth user used for authentication in nginx is ditto with password ditto
+  # add more users to the basicAuthUsers array or replace the default user
+  basicAuthUsers:
+    ditto:
+      user: ditto
+      password: ditto
```

Comment:
we should add an option to also be able to provide the already hashed user/password  list here, so that you don't have to put cleartext passwords in your Helm config

proposal: 
* support 2 different types of username/password combinations to configure:
   * `basicAuthUsers` (e.g. being the cleartext password like you already configured it)
   * `hashedBasicAuthUsers` (a list of already by `htpasswd` hashed username/password lines)

Reply:
yes sure this is a good idea. I can add the `hashedBasicAuthUsers` type. but then I suggest that it is only possible to configure one of the types. because otherwise it will get really complicated. @thjaeckle wdyt?

Reply:
yes, that should be sufficient - either one or the other 👍

Reply:
@thjaeckle I have added a hashedBasicAuthUsers section and now it is possible to use both approaches to configure basic auth users.